package.json

```diff
@@ -201,7 +201,7 @@
     "manualTestProxyJest": "jest mmExtIntegration.test.ts",
     "release:linux": "vite build && electron-builder -p always --linux deb AppImage rpm pacman tar.xz",
```

Comment:
I think we should also remove AppImage (since it has a bug with the mm extension that loses configuration on restart)